airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -47,7 +47,7 @@ acceptance_tests:
             ["airbytehq/integration-test", "907296275", "created_at"]
           issue_events: ["airbytehq/integration-test", "created_at"]
           issue_milestones: ["airbytehq/integration-test", "updated_at"]
-          issue_reactions: ["airbytehq/integration-test", "11", "created_at"]
+          issue_reactions: ["airbytehq/integration-test", "created_at"]
```

Comment:
Does this change imply that the format of state changed for an incremental stream?  If so, that's probably a major (breaking) semver change

Reply:
yes state format changed, state object simplified
Can you advice how to be in such cases? Are you proposing to increase version number 0.3.11 -> 0.4.0?

Thank you

Reply:
To answer the question, the user experience needs to be considered:
1. If the sync will continue as it has in the past (maintain it's incremental position) - patch change (no user intervention needeed)
2. If the sync will succeed, but require a full import again because the state format is changed - minor change (no user intervention needed, but a slow sync will happen)
3. If the sync will fail until the user takes action or the records returned are changed - major change.

I think we are in category 2, like you suggest - is that correct?

Reply:
answer - 2
because the state format is changed 
it will re-read all issue-reaction records again from "start_date"

Reply:
Thanks! Then as you suggest, this should be v0.4.0 of the connector.
We are trying to get good at semver versioning of connectors from the user's point of view so we can automate warnings and breaking change announcements in the app.

Reply:
I have bumped to 0.4.0

Reply:
@evantahler can you please look can we proceed with this PR?